ansible/roles/ssh/tasks/main.yaml

```diff
@@ -22,5 +22,11 @@
     - { src: "{{jenkins_public_ssh_key_file_path}}",
         dest: "{{jenkins_home_dir}}/.ssh/authorized_keys",
         owner: jenkins, group: jenkins, mode: "0600" }
+
+- name: Add known keys for remote hosts
+  copy:
+    src={{known_hosts_file_path}} dest="{{jenkins_home_dir}}/.ssh/known_hosts"
+    owner=jenkins group=jenkins
+  
```

Comment:
This role is executed by both master and slave playbooks and will fail on the master because there is no value for `known_hosts_file_path` when it is executed.
Move it to the end of the [builds role](https://github.com/open-power-host-os/infrastructure/blob/master/ansible/roles/builds/tasks/main.yaml), which is where the SSH keys for those servers are actually configured.